CHANGELOG.md

```diff
@@ -0,0 +1,227 @@
+
+#Changelog
+
+### 1.27 (Aug 26, 2018)
+
+* [JENKINS-51876](https://issues.jenkins-ci.org/browse/JENKINS-51876) Update Core and Java version, remove deprecated code
+* [JENKINS-52660](https://issues.jenkins-ci.org/browse/JENKINS-52660) Allow to disable TCP_NODELAY
+* [JENKINS-42840](https://issues.jenkins-ci.org/browse/JENKINS-42840) Slave to agent Renaming: SSH Slaves Plugin
+* [JENKINS-44111](https://issues.jenkins-ci.org/browse/JENKINS-44111) Enable Remoting work dir by default in SSH Slaves Plugin
+* [JENKINS-52613](https://issues.jenkins-ci.org/browse/JENKINS-52613), [JENKINS-52739](https://issues.jenkins-ci.org/browse/JENKINS-52739) configure 10 retries, 15 seconds between retries, and a timeout of about 210 seconds to connect to the agent.
+* [JENKINS-47586](https://issues.jenkins-ci.org/browse/JENKINS-47586) Removes JDK installer
+* [JENKINS-37152](https://issues.jenkins-ci.org/browse/JENKINS-37152) Support Win32-OpenSSH
+* [JENKINS-49235](https://issues.jenkins-ci.org/browse/JENKINS-49235) Allow disable credentials tracking
+* [JENKINS-53174](https://issues.jenkins-ci.org/browse/JENKINS-53174) Improve documentation
+
+#### Breaking changes
+
+* Requires Jekins 2.73.3
+* Requires JDK/JRE 8 installed on agents
+* The slave.jar file copied on the agent work folder change name to remoting.jar
+
+### 1.26 (Feb 26, 2018)
+* JENKINS-49607 - Report the required Java version when the plugin cannot find Java on the agent
```

Comment:
Would be nice to add links to the tickets. Could probably use `sed` or something?